Document/0x06h-Testing-Platform-Interaction.md

```diff
@@ -287,14 +296,15 @@ In order to address these attack vectors, the outcome of the following checks sh
 - V6.6: "WebViews are configured to allow only the minimum set of protocol handlers required (ideally, only https is supported). Potentially dangerous handlers, such as file, tel and app-id, are disabled."
 
 ##### CWE
--- TODO [Add relevant CWE for "Testing WebView Protocol Handlers"] --
+N/A
```

Comment:
[CWE-939: Improper Authorization in Handler for Custom URL Scheme](https://cwe.mitre.org/data/definitions/939.html)